docs/source/example_input/boosted_frame_script.py

```diff
@@ -137,19 +137,26 @@ def dens_func( z, r ):
 # Velocity of the Galilean frame (for suppression of the NCI)
 v_comoving = - c * np.sqrt( 1. - 1./boost.gamma0**2 )
 
-# The diagnostics
-diag_period = 50        # Period of the diagnostics in number of timesteps
-# Whether to write the fields in the lab frame
-Ntot_snapshot_lab = 15
-dt_snapshot_lab = 10*(zmax-zmin)/c
-track_bunch = False  # Whether to tag and track the particles of the bunch
-
 # The interaction length of the simulation (meters)
 L_interact = (p_zmax-p_zmin) # the plasma length
 # Interaction time (seconds) (to calculate number of PIC iterations)
 T_interact = boost.interaction_time( L_interact, (zmax-zmin), v_window )
 # (i.e. the time it takes for the moving window to slide across the plasma)
 
+## The diagnostics
+
+# Number of diagnostics to output (including first output at t=0)
+N_diag = 10+1
+# Time interval between diagnostics (first at t=0, last at t=T_interact)
+dt_diag = (L_interact + (zmax-zmin)) / v_window / (N_diag - 1)
+# Period of the (boosted frame) diagnostics in timesteps
+diag_period = T_interact / (N_diag - 1)
```

Comment:
Hm, here `diag_period` should be `int( T_interact / ((N_diag - 1) * dt_boost) )`, but `dt_boost` is not very easy to obtain (we get it below, by using `sim.dt`, but this is not defined yet here) ; alternatively, we could add a method `boost.time` to the `BoostConverter`. @MKirchen @hightower8083 Any opinion?

Reply:
oups, haven’t noticed absence of dt in diag_period definition.. 
i think that adding an explicit time coverter for dt_boost calculation is a good idea.

Reply:
Sorry, my fault. I agree that we should add a boost.time() method.